delete_book.php

```diff
@@ -0,0 +1,15 @@
+<?php
+include 'database.php';
+
+if (isset($_GET['id'])) {
+    $id = $_GET['id'];
+
+    $query = "DELETE FROM books WHERE id = $id";
+    if (mysqli_query($connection, $query)) {
+        echo "Book deleted successfully!";
+        header("Location: list_books.php");
+    } else {
+        echo "Error: " . mysqli_error($connection);
```

Comment:
Observations: The SQL query is vulnerable to SQL injection due to the direct inclusion of user input.

Recommendations:
- Use prepared statements to prevent SQL injection.